### DISCUSSION TITLE
Gitignore problem after hitting "back"

### DISCUSSION THREAD

**Discussion by vicmarconi** [community/community#3907] [Repositories]
Hi,

I'd like to inform a behavior that I think is wrong:

1) Go "Create repo" page, fill fields, add a gitignore for a language (ie Python)
2) Click "create repository" (then its created and forwarded to its page)
3) Click "back" (browser)

The error is: even tough gitignore is checked, "None" is shown on the field. But it is indeed selected (there is a "V").

<img width="367" alt="Screen Shot 2021-05-07 at 05 25 12" src="https://user-images.githubusercontent.com/1854108/117421119-9cbfc980-aef4-11eb-94a7-5ff9f12ee55e.png">

**Comment by akay222** [comment]
I do not know where to go to pull that up all I can pull up are the
exported emails I asked for because when I logged on I saw that they were
deleted

**Comment by akay222** [comment]
Ok, I'll check it out.
